### PR TITLE
Renamed package name from java to kotlin for kt files

### LINKED ISSUE #13: Classes should be moved over from src/main/java to src/main/kotlin

### PR DESCRIPTION
**Pull request by tharakadesilva**
Resolves #13

### CODE REVIEW

File: generators/server/files.js [hunk 1]
```diff
@@ -313,21 +320,21 @@ function writeFiles() {
             }
 
             if (this.authenticationType === 'session') {
-                this.template(rewriteDir(`${SERVER_MAIN_SRC_DIR}package/domain/PersistentToken.kt.ejs`), `${javaDir}domain/PersistentToken.kt`);
+                this.template(rewriteDir(`${SERVER_MAIN_SRC_KOTLIN_DIR}package/domain/PersistentToken.kt.ejs`), `${kotlinDir}domain/PersistentToken.kt`);
```
Comment (sendilkumarn) [inline generators/server/files.js]: can this be added to the `rewriteDir` function above ?

**Comment by tharakadesilva** [comment]
@sendilkumarn its ready for another review.

**Comment by deepu105** [comment]
Gradle uses different directory for each JVM language, so might have to do
something in the build settings


Thanks & Regards,
Deepu

**Comment by sendilkumarn** [comment]
ofcourse the problem is in maven build. We should call the maven experts here. 😉

**Comment by tharakadesilva** [comment]
I'll revert this for now. Test maven and create a new branch.

**Pull request merged**